### PR TITLE
Seekrets install - download and enable rulesets

### PR DESCRIPTION
**Pull request by alain-hoang**
This PR enables downloading and enabling of rulesets in the seekrets-install script.

### CODE REVIEW

**Comment by alain-hoang** [comment]
The failures that are occurring in #69 are a result of a missing dependency.  See #75 for more information regarding this.  This PR also includes changes to address warnings issued from travis however the build failure requires a change in upstream.  When the dependency has been fixed at upstream then the missing dependency build failures should stop occurring.

**Comment by rogeruiz** [comment]
@alain-hoang this PR is going to merge all of the changes in to `master` when it should be pointing to `seekret` instead.

**Comment by alain-hoang** [comment]
Thank you @rogeruiz for ensuring this pull request goes to the proper branch!   I was under the impression that branching from the the intended branch `seekret` would work but not in this case.

File: seekrets-install [hunk 1]
```diff
@@ -51,16 +51,18 @@ if toolversion git '>=' 2.9.1; then
...
 	  && curl -# -o "${rule}" "${SEEKRET_RULES_URL}/${rule}")
   done
   fancy_echo "Configuring Seekret"
   mkdir -p "${HOME}/.git-support/hooks"
   git config --global core.hooksPath "${HOME}/.git-support/hooks"
   git seekret --global config --init
-  # TODO: enable rules here...
   git seekret --global hook --enable-all
+  # Activate all installed rules
+  git seekret rules | grep '\[ \] ' | awk ' { print $3 } ' | xargs -n 1 git seekret --global rules --enable
```
Comment (LinuxBozo) [inline seekrets-install]: Given https://github.com/apuigsech/git-seekret/pull/15 this can be simplified.

**Pull request merged**